resources/js/mosaic.overlay.js

```diff
@@ -50,7 +51,8 @@ export default class Overlay {
         const ajax_url_parts = self.ajax_edit_url().split("?");
         self.modal.on("formActionSuccess", (e) => {
             $("#content-core", $(e.target)).load(
-                `${ajax_url_parts[0]} #content-core > *`, ajax_url_parts[1], () => {});
+                `${ajax_url_parts[0]} #content-core > *`, ajax_url_parts[1], () => { });
+            registry.scan("#content-core");
```

Comment:
This was part of another investigation but doesn't work ... line 55 and the import above should be removed again.